### ISSUE TITLE
Alternative choice :)

### ISSUE THREAD

**Issue by LazyZhu** [m13253/geodns-injector#1]
Nice utility, try it later :)
BTW, i'm using modified miekg's [coredns](https://github.com/coredns/coredns) with [rewrite](https://github.com/coredns/coredns/tree/master/plugin/rewrite) plugin to change EDNS0_SUBNET on the fly.

**Comment by m13253** [comment]
I have once used CoreDNS's rewrite plugin.
But that did not suit my needs: I need to keep IP unchanged if from a certain country, and rewrite IP otherwise.
This is useful for optimizing for different ISPs.

**Comment by LazyZhu** [comment]
Yes, original CoreDNS can't change the ip address of EDNS0_SUBNET but the netmask.
Although, CoreDNS has other useful plugins like [proxy](https://github.com/coredns/coredns/tree/master/plugin/proxy), [forward](https://github.com/coredns/coredns/tree/master/plugin/forward), load balance, etc.
